graduated/gitops-argocd-sync/step.yaml

```
@@ -1,7 +1,7 @@
 kind: step-type
 metadata:
   name: gitops-argocd-sync
-  version: 1.4.5
+  version: 1.4.6
```

@pysarenko-bohdan let's bump minor, it's a new feature
```suggestion
  version: 1.5.0
```